bin/distmap

```diff
@@ -20,124 +20,166 @@ use DownloadTrimmedRead;
 use DataCleanup;
 use ReadToolsUtils;
 
-my $script = "distmap";
+# current version
+use constant VERSION => "3.1.0-alpha";
 
-my $version = "3.1.0-alpha";
-
-### This number need to define. How many processors Hadoop can use on each nodes.
+## TODO - make a constant!!
+## TODO - should be exposed? or maybe depends on other variable (e.g., priority?)
+### This number need to define. How many processors Hadoop can use on each nodes
 my $number_of_processors = 3; #18;
```

Comment:
I think this is actually unused.

Reply:
It is actually not unused: it adds the processors to the map of arguments, and some mappers use the values (passed from Utility to the mapper script).

Reply:
I open #81 for tracking this.